src/Microsoft.Framework.Runtime.Abstractions/project.json

```diff
@@ -2,6 +2,12 @@
     "version": "1.0.0-*",
     "description": "ASP.NET 5 interfaces to take advantage of capabilities exposed by the runtime.",
     "compilationOptions": { "warningsAsErrors": true },
+    "dependencies": {
+        "Microsoft.Framework.HashCodeCombiner.Sources": {
```

Comment:
I suggest you take a look at using .editorconfig files to control tab vs. spaces and indent size